core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -0,0 +1,405 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.meta;
+
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.PropertyKey;
+import alluxio.Server;
+import alluxio.clock.SystemClock;
+import alluxio.collections.IndexDefinition;
+import alluxio.collections.IndexedSet;
+import alluxio.exception.ExceptionMessage;
+import alluxio.exception.status.NotFoundException;
+import alluxio.heartbeat.HeartbeatContext;
+import alluxio.heartbeat.HeartbeatExecutor;
+import alluxio.heartbeat.HeartbeatThread;
+import alluxio.master.AbstractMaster;
+import alluxio.master.MasterClientConfig;
+import alluxio.master.MasterContext;
+import alluxio.master.block.BlockMaster;
+import alluxio.proto.journal.Journal.JournalEntry;
+import alluxio.retry.ExponentialTimeBoundedRetry;
+import alluxio.retry.RetryUtils;
+import alluxio.thrift.MetaCommand;
+import alluxio.thrift.MetaMasterClientService;
+import alluxio.thrift.MetaMasterMasterService;
+import alluxio.thrift.RegisterMasterTOptions;
+import alluxio.util.IdUtils;
+import alluxio.util.executor.ExecutorServiceFactories;
+import alluxio.util.executor.ExecutorServiceFactory;
+import alluxio.util.network.NetworkAddressUtils;
+import alluxio.wire.ConfigProperty;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
+import org.apache.thrift.TProcessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.time.Clock;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Collectors;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * The master that handles all metadata management.
```

Comment:
file system master manages file metadata, block master manages block metadata

Reply:
Changed to ``` that master that handles Alluxio metadata management```, thanks

Reply:
the point is that all masters manage metadata, so saying this master manages metadata isn't very meaningful